src/coguard_cli/__main__.py

```diff
@@ -94,8 +94,9 @@ def main():
         '--output-format',
         type=str,
         dest='output_format',
-        default='formatted',
-        choices=['formatted', 'json', 'sarif'],
+        nargs='*',
```

Comment:
Upon further consideration, we should make this a string. Otherwise, there might be backwards compatibility issues.